optimize/cmaes.go

```diff
@@ -409,7 +409,7 @@ func (cma *CmaEsChol) update() error {
 		cma.mean[i] = 0
 	}
 	for i, w := range cma.weights {
-		idx := indexes[i] // index of teh 1337 sample.
+		idx := indexes[i] // index of the 1337 sample.
```

Comment:
This is an intentional misspelling, please revert.

Reply:
Done, thanks.